### PR TITLE
Trying to fix some failed tests

### PR DESCRIPTION
**Pull request by zerc**

### CODE REVIEW

File: tests/integration/test_api_resources.py [hunk 1]
```diff
@@ -287,13 +287,6 @@ def tearDownClass(cls):
         conf.pub_key = None
         conf.secret = None
 
-    def test_errors(self):
```
Comment (zerc) [inline tests/integration/test_api_resources.py]: @dmitry-mukhin @homm if we use mock then i don't see purpose of this test

File: tests/integration/test_api_resources.py [hunk 2]
```diff
@@ -301,13 +294,15 @@ def test_local_copy(self):
         response = self.f.copy(effects='resize/50x/')
         self.assertEqual('file', response['type'])
 
-    def test_remote_copy(self):
+    @mock.patch('pyuploadcare.api_resources.rest_request')
+    def test_remote_copy(self, rest_request):
```
Comment (zerc) [inline tests/integration/test_api_resources.py]: This too but i trying :)

File: tests/integration/test_api_resources.py (hunk 1 above)
Comment (dmitry-mukhin) [inline tests/integration/test_api_resources.py]: there is none

File: tests/integration/test_api_resources.py (hunk 2 above)
Comment (dmitry-mukhin) [inline tests/integration/test_api_resources.py]: replace this with a test that checks the request payload

Comment (zerc) [inline tests/integration/test_api_resources.py]: Btw we already testing it here: https://github.com/uploadcare/pyuploadcare/blob/master/tests/functional/test_api_resources.py#L167-L198

Then i think we can drop the `test_remote_copy(self, rest_request)` now isn't?

**Pull request merged**